node/src/main/scala/coop/rchain/node/diagnostics/client/Runtime.scala

```diff
@@ -35,18 +35,6 @@ object Runtime {
       _       <- ConsoleIO[F].close
     } yield ()
 
-    for {
-      result <- program.attempt
-      _ <- result match {
-            case Left(ex) => ConsoleIO[F].println(s"Error: ${processError(ex).getMessage}")
-            case _        => ().pure[F]
-          }
-    } yield ()
-  }
```

Comment:
This seems like an unrelated change for `Remove everywhere ErrorHandler as typeclass constraint`. Is the delete intentional?

Reply:
yes, program is task so you no longer call attempt on it. you will not get those `CommErrors` return here.